pkg/pool-weighted/contracts/test/Hevm.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+// solhint-disable
+
+interface Hevm {
```

Comment:
Should this just go up in /interfaces?

Reply:
Looks like I was being dumb and not importing the right test helper so we don't need this at all.